etc/servo_automation_screenshot.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Mar 26 20:08:25 2018
+
+@author: Pranshu Sinha, Abhay Soni, Aayushi Agrawal
+"""
+
+"""
+The below program is intended to test rendering mismatches in servo based on random web pages generated by the web page fuzzer.
+
+Here is the breakdown of how our code works:
+*   A session is started on localhost:7002
+*   The randomly generated webpage's (html files) data is sent as JSON to this session
+*   Using curl request, we load the html files for this session ID based on the session we just created
+*   For each html file it renders, it takes the screenshot after rendering which is saved in the current working directory in the format "testing#.png"
+"""
+import os
+import subprocess
+import json
+import requests
+import start_servo
+import time
+
+#The below function is used to start servo on localhost:7002
+start_servo.start_servo()
+
+#Since servo takes time to load and to get the session ID for subsequent steps, we added a sleep for 60 seconds
+time.sleep(60)
+
+#The program assumes that the html files to be rendered in servo are present in the same current working directory in the format "file#.html"
```

Comment:
Let's accept URLs or file names as command line arguments rather than making this assumption.